boards/arm/mimxrt685_evk/doc/index.rst

```diff
@@ -227,6 +227,20 @@ configured by default to use the LPC-Link2.
 
 .. tabs::
 
+    .. group-tab:: LinkServer CMSIS-DAP
+
+        1. Install the :ref:`linkserver-debug-host-tools` and make sure they are in your
+           search path.  LinkServer works with the default CMSIS-DAP firmware included in
+           the on-board debugger.
+        2. Make sure the jumpers JP17, JP18 and JP19 are installed.
+
+        linkserver is the default runner for this board
```

Comment:
Since the default runner will no longer be JLink, this doc page needs more changes:
1) The tab `LPCLink2 JLink Onboard` should include details needing `-r jlink` parameter
2) The tab `JLink External` should include details needing `-r jlink` parameter
3) In the `Flashing` section, this statement needs to change to using LinkServer as default
> This example uses the [J-Link Debug Host Tools](https://docs.zephyrproject.org/latest/develop/flash_debug/host-tools.html#jlink-debug-host-tools) as default.
4) In the `Debugging` section, this statement needs to change to using LinkServer as default
> This example uses the [J-Link Debug Host Tools](https://docs.zephyrproject.org/latest/develop/flash_debug/host-tools.html#jlink-debug-host-tools) as default.